safe-global/safe-wallet-web
# Feat: create new outreach campaign targeting all safes [SW-638]

## What it solves

Resolves: https://www.notion.so/safe-global/76f265123c0a4ad680947ce489804342?v=f9f164524ec4464286fdc38f31589d0c&p=1528180fe57380d8889de953b3fa0a66&pm=s

## How this PR fixes it
- adds new active outreach with a link to the new survey in typeform
- passes safe address, signer address, and chain id to the form to be recorded with the answers.

## How to test it
- Open a safe. The outreach popup should be shown.
- clicking the CTA should open the survey in typeform. Upon finishing the survey, check that the safe addreess, signer address, and chain ID are included in the answers.
- After clicking through to the survey, the popup should not be shown for that safe/signer combination again.

## Screenshots
![image](https://github.com/user-attachments/assets/ef77bcfa-2fe9-4509-81df-3a89c1d6b78d)

## Checklist
